src/google-maps/map-polyline/map-polyline.spec.ts

```diff
@@ -0,0 +1,145 @@
+import {Component} from '@angular/core';
+import {async, TestBed} from '@angular/core/testing';
+import {By} from '@angular/platform-browser';
+
+import {DEFAULT_OPTIONS, UpdatedGoogleMap} from '../google-map/google-map';
+import {GoogleMapsModule} from '../google-maps-module';
+import {
+  createMapConstructorSpy,
+  createMapSpy,
+  createPolylineConstructorSpy,
+  createPolylineSpy,
+  TestingWindow,
+} from '../testing/fake-google-map-utils';
+
+import {MapPolyline} from './map-polyline';
+
+const POLYLINE_PATH: google.maps.LatLngLiteral[] =
+    [{lat: 25, lng: 26}, {lat: 26, lng: 27}, {lat: 30, lng: 34}];
+const POLYLINE_OPTIONS: google.maps.PolylineOptions = {
```

Comment:
It might be a good idea to put these inside the `describe` an re-assign them in the `beforeEach`, otherwise they might end up retaining state between tests if they get mutated.

Reply:
Done.